CHANGES.md

```diff
@@ -4,12 +4,14 @@
 
 ### Major Changes
 
-* `augur.io.read_metadata` (used by export, filter, frequencies, refine, and traits): Previously, this supported any arbitrary delimiters for the metadata. It is now restricted to CSV and TSV, which are the officially supported formats for all Augur subcommands that use this function. [#812][] (@victorlin)
+* export, filter, frequencies, refine, traits: From versions 10.0.0 through 21.1.0, arbitrary delimiters for `--metadata` were supported due to internal implementation differences from the advertised CSV and TSV support. Starting with this version, non-CSV/TSV files will no longer be supported by default. To adjust for this breaking change, specify custom delimiters with the new `--metadata-valid-delimiters` flag. [#1196][] (@victorlin)
```

Comment:
Ah, that'd be fine. I'd been wary of using an unqualified "delimiters" since the initial implication to me is that a mix of delimiters would be accepted for a single file (which is horrifying)… but I hadn't realized we had precedent with "id columns". The consistency and shorter name is good, and hopefully an initial horror like mine is quickly replaced by more reasonable understanding. ;-)

Reply:
Renamed to use "delimiters" in [force-push](https://github.com/nextstrain/augur/compare/2bbf9288fdb38e969bfd838a3f9aaa6287973bd9..1ac4649bc6044bc1333b0c07a8af70c815d451b5).

Also pushed 4e960edccf4f53430d68d3f01757f222bc11cc56 to address your horror (which I also had thought about).